src/platform/assets/composables/useAssetBrowser.ts

```diff
@@ -198,10 +198,6 @@ export function useAssetBrowser(assets: AssetItem[] = []) {
     assetId: string,
     onSelect?: (filename: string) => void
   ): Promise<void> {
-    if (import.meta.env.DEV) {
-      console.debug('Asset selected:', assetId)
-    }
-
```

Comment:
@arjansingh This okay to remove now that things are actually wired up?

Reply:
oh yeah, get rid of this, thank you. oversight on my part.